src/components/common/Tabs/Tabs.module.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../../styles/settings.scss";
+
+.tabsHeader {
+    display: flex;
+    flex-direction: row;
+    padding-left: 0;
+    margin-bottom: 0;
+    list-style: none; 
+    width: 100%;
+
+    .tabsItem {
+        width: auto;
+        padding: 1rem;
+        border: 1px solid $global-border-color;
```

Comment:
zastosowaleś ogolny border dookoła i przez to bordery sie sumuja. 
![cR tabsy](https://user-images.githubusercontent.com/75266912/116903848-437a4080-ac3d-11eb-8c4d-0d32b02049ae.JPG)
 realnie na tabach by zrobił tylko border-right i border-bottom, potem bottom i tak sie na active wyłączy :)

Reply:
Poszedł hotFix do tego :) dzięki za uwagę !